sockeye/arguments.py

```diff
@@ -915,7 +900,7 @@ def add_training_args(params):
 
     train_params.add_argument('--seed',
                               type=int,
-                              default=13,
+                              default=1,
```

Comment:
Same here, I would suggest going back to `13`.

Reply:
The idea is that `1` would be more intuitive and perhaps `0` would be even better.  Again since we aren't backward compatible anyway, we can pick defaults that are either stronger or more intuitive.

Reply:
ok, as long as system tests still pass, we'd should be fine here.